fedora_messaging/cli.py

```diff
@@ -452,3 +458,34 @@ def record(exchange, queue_name, routing_key, app_name, limit, file):
     _consume(
         exchange, queue_name, routing_key, messages_recorder.collect_message, app_name
     )
+
+
+URL_TEMPLATE = "https://apps.fedoraproject.org/datagrepper/id?id={}&is_raw=true"
+
+
+@cli.command()
+@click.argument("message_id")
+@click.option("--datagrepper-url", help=_datagrepper_url, default=URL_TEMPLATE)
+def replay(message_id, datagrep_url):
```

Comment:
Hmm, does this actually work? I'm surprised that click is able to match the `--datagrepper-url` option name to the `datagrep_url` variable name. If not I would recommend renaming the variable.

Reply:
No worries, but a unit test would have caught that ;-)